imports/ui/StudentQuestionResultsListItem.jsx

```diff
@@ -11,15 +11,15 @@ import { WysiwygHelper } from '../wysiwyg-helpers'
 import { Responses } from '../api/responses'
 import { _ } from 'underscore'
 
-export class _StudentQuestionResultsListItem extends Component {
+export class StudentQuestionResultsListItem extends Component {
 
   componentWillMount () {
     MathJax.Hub.Queue(['Typeset', MathJax.Hub])
   }
 
   render () {
     const q = this.props.question
-    const unique = _.uniq(q.studentResponses, 'questionId')
+    const unique = _.where(q.studentResponses, { studentUserId: Meteor.userId() })
```

Comment:
Shouldn't studentResponses already be limited in the publication to that users's ID, since we don't want students to see other's responses? Ideally, we don't have to specify Meteor.userId() anywhere on the client code.